### PR TITLE
Add journeypeople and team member minutes from 2019-02-28 meeting

### PR DESCRIPTION
**Pull request by spastorino**
r? @nikomatsakis 

/cc @davidtwco

### CODE REVIEW

**Comment by spastorino** [comment]
> This is looking great @spastorino! I've left a bunch of minor nits, I hope you don't mind, they are mostly small grammar or wording tweaks to tighten things up a bit.

I don't mind at all. Thanks for that because it helps me to improve my english :).

File: working-groups/meta/NOTES.md [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+
+- What the journeyperson role means?
+  - You know something about the compiler?
+  - People willing to commit some time to help run things
+  - Should a journeypeople be able to onboard more compiler contributors?
+  - r+ rights?
+  - Start to do reviews?
+  - What should a journeypeople do?
+
+- What does a compiler team member means?
```
[the remark this answers is not in the document]
Comment (spastorino) [reply working-groups/meta/NOTES.md]: I ended thinking that both things needed discussion. I'm happy to remove this if it's not the case.

**Comment by spastorino** [comment]
@davidtwco fixed a lot of your comments.

File: working-groups/meta/NOTES.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidtwco) [reply working-groups/meta/NOTES.md]: I think most of the relevant things that this question would surface will also come out of the "What's the difference between journeypeople and team members?" question.

[the remark this answers is not in the document]
Comment (nikomatsakis) [reply working-groups/meta/NOTES.md]: My sense here is that I don't care :P that is, I don't think having the heading does any harm, but I also think that @davidtwco is correct that it overlaps the next heading. Also, just a nit, but the grammatical structure is confusing -- perhaps "What does it mean to be a member of the compiler team?" would be better, but that's also kind of vague. Maybe "What are the qualifications and expectations for a member of the compiler team" or something else would be more specific.

File: working-groups/meta/NOTES.md [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+  - What is the role of a journeyperson in onboarding new compiler contributors?
+  - r+ rights?
+  - Start to do reviews?
+  - What responsibilities should a journeyperson have?
+  - Are journeypeople members of the team or is a journeyperson role considered a stepping stone toward being a member of the team?
+
+
+- What are the qualifications and expectations for a member of the compiler team?
+
+- What's the difference between journeypeople and team members?
```
Comment (Centril) [inline working-groups/meta/NOTES.md]: rfcbot powers seems like an important distinction? Notably, if you have too many team members, rfcbot will likely get stuck in limbo on all sorts of issues and decisions that need to be made.

Comment (nikomatsakis) [reply working-groups/meta/NOTES.md]: (while potentially true, this is meant to be minutes from the meeting that occurred)

Comment (Centril) [reply working-groups/meta/NOTES.md]: Oh; right... forgot ^^

**Pull request merged**